dockerfiles/docker-compose.yml

```diff
@@ -93,58 +93,10 @@ services:
       - 30303
     command: [
       "--verbosity", "2",
-      "--rpc",
-      "--rpcaddr", "0.0.0.0",
+      "--http",
```

Comment:
these are updates based on warnings when running geth -  the `rpc` flags are being deprecated in favor of `http`.

Reply:
👍